alembic/versions/0f1f62adc91c_notice_types.py

```diff
@@ -0,0 +1,337 @@
+            'SWIFT_XRT_IMAGE_PROC',
+            'SWIFT_UVOT_DBURST_PROC',
+            'SWIFT_UVOT_FCHART_PROC',
+            'SWIFT_UVOT_POS',
+            'SWIFT_BAT_GRB_POS_TEST',
+            'SWIFT_POINTDIR',
+            'SWIFT_BAT_TRANS',
+            'SWIFT_XRT_THRESHPIX',
+            'SWIFT_XRT_THRESHPIX_PROC',
+            'SWIFT_XRT_SPER',
+            'SWIFT_XRT_SPER_PROC',
+            'SWIFT_UVOT_POS_NACK',
+            'SWIFT_BAT_ALARM_SHORT',
+            'SWIFT_BAT_ALARM_LONG',
+            'SWIFT_UVOT_EMERGENCY',
+            'SWIFT_XRT_EMERGENCY',
+            'SWIFT_FOM_PPT_ARG_ERR',
+            'SWIFT_FOM_SAFE_POINT',
+            'SWIFT_FOM_SLEW_ABORT',
+            'SWIFT_BAT_QL_POS',
+            'SWIFT_BAT_SUB_THRESHOLD',
+            'SWIFT_BAT_SLEW_POS',
+            'AGILE_GRB_WAKEUP',
+            'AGILE_GRB_GROUND',
+            'AGILE_GRB_REFINED',
+            'SWIFT_ACTUAL_POINTDIR',
+            'AGILE_MCAL_ALERT',
+            'AGILE_POINTDIR',
+            'AGILE_TRANS',
+            'AGILE_GRB_POS_TEST',
+            'FERMI_GBM_ALERT',
+            'FERMI_GBM_FLT_POS',
+            'FERMI_GBM_GND_POS',
+            'FERMI_GBM_LC',
+            'FERMI_GBM_GND_INTERNAL',
+            'FERMI_GBM_FIN_POS',
+            'FERMI_GBM_ALERT_INTERNAL',
+            'FERMI_GBM_FLT_INTERNAL',
+            'FERMI_GBM_TRANS',
+            'FERMI_GBM_POS_TEST',
+            'FERMI_LAT_POS_INI',
+            'FERMI_LAT_POS_UPD',
+            'FERMI_LAT_POS_DIAG',
+            'FERMI_LAT_TRANS',
+            'FERMI_LAT_POS_TEST',
+            'FERMI_LAT_MONITOR',
+            'FERMI_SC_SLEW',
+            'FERMI_LAT_GND',
+            'FERMI_LAT_OFFLINE',
+            'FERMI_POINTDIR',
+            'SIMBADNED',
+            'FERMI_GBM_SUBTHRESH',
+            'SWIFT_BAT_MONITOR',
+            'MAXI_UNKNOWN',
+            'MAXI_KNOWN',
+            'MAXI_TEST',
+            'OGLE',
+            'CBAT',
+            'MOA',
+            'SWIFT_BAT_SUBSUB',
+            'SWIFT_BAT_KNOWN_SRC',
+            'VOE_11_IM_ALIVE',
+            'VOE_20_IM_ALIVE',
+            'FERMI_SC_SLEW_INTERNAL',
+            'COINCIDENCE',
+            'FERMI_GBM_FIN_INTERNAL',
+            'SUZAKU_LC',
+            'SNEWS',
+            'LVC_PRELIMINARY',
+            'LVC_INITIAL',
+            'LVC_UPDATE',
+            'LVC_TEST',
+            'LVC_COUNTERPART',
+            'AMON_ICECUBE_COINC',
+            'AMON_ICECUBE_HESE',
+            'CALET_GBM_FLT_LC',
+            'CALET_GBM_GND_LC',
+            'LVC_EARLY_WARNING',
+            'LVC_RETRACTION',
+            'GWHEN_COINC',
+            'AMON_ICECUBE_EHE',
+            'HAWC_BURST_MONITOR',
+            'AMON_NU_EM_COINC',
+            'ICECUBE_ASTROTRACK_GOLD',
+            'ICECUBE_ASTROTRACK_BRONZE',
+            'SK_SN',
+            'ICECUBE_CASCADE',
+            name='noticetype',
+        ),
+        type_=sa.String(),
+        existing_nullable=True,
```

Comment:
What is this migration doing? I get from the models that you don't use the ENUM from pygcn anymore, so why does it not just drop the enum here? Just curious

Reply:
But there is still a column. So it's an alter I guess.